Update consensus mechanisms developer doc intro (#4934)
ethereum/ethereum-org-website: src/content/developers/docs/consensus-mechanisms/index.md

## What is consensus? {#what-is-consensus}

> By consensus, we mean that a general agreement has been reached. Consider a group of few people going to the cinema. If there are no principal disagreements on a film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.

Reviewer: ```suggestion
By consensus, we mean that a general agreement has been reached. Consider a group of people going to the cinema. If there is not a disagreement on a proposed choice of film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

> By consensus, we mean that a general agreement has been reached. Consider a group of few people going to the cinema. If there are no principal disagreements on a film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.

Reviewer: Choice of word improvements: 

```suggestion
By consensus, we mean that a general agreement has been reached. Consider a group of people going to the cinema. If there is not a disagreement on a proposed choice of film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

Reviewer: @elshigori What is this supposed to say: "Participants either get the group on their side, either side with the group."

Author: It's just an outline of options for any consensus to move forward: 
* propagate your position among peers, 
* adjust your position to others, 
* or split.
